### ISSUE TITLE
It throws a UnsupportedOperationException when there is no default constructor.

### ISSUE THREAD

**Issue by vishwa-** [keepcosmos/beanmother#22]
I am trying to create an instance of a object that is annotated with `@Builder` from `lombok.Builder`
It throws an java.lang.UnsupportedOperationException saying that the object has no default constructor.

Is it possible to add a new feature to create the object with All args constructor ?

**Comment by keepcosmos** [comment]
You should try lombok builder extension. please try [Builder Converter](https://github.com/keepcosmos/beanmother/tree/master/beanmother-builder-converter)

**Comment by vishwa-** [comment]
Is builder converter released for GA?

**Comment by keepcosmos** [comment]
@josejuanmontiel Could you answer?

**Comment by josejuanmontiel** [comment]
yes @keepcosmos i can... the code it's meged ;) but no released :( under 0.7.6

**Comment by keepcosmos** [comment]
Ooops. sorry @josejuanmontiel I will release as soon as possible

**Comment by keepcosmos** [comment]
I just release new version (0.8.0). 

@vishwa- Please check few minutes later (because of maven centeral cache)

@josejuanmontiel Thanks and sorry for late :)
